src/app/categories/bookmarks/bookmarks.js

```diff
@@ -17,7 +17,8 @@ angular.module('categories.bookmarks', [
             })
         ;
     })
-    .controller('BookmarksCtrl', function ($scope,$stateParams  ) {
-        $scope.currentCategoryName = $stateParams.category;
+    .controller('BookmarksCtrl', function ($stateParams) {
```

Comment:
This variable should reflect the name of the controller so main should be `var bookmark = this;`
Line 14 should say BookmarkCtrl as bookmark